IBA/utils.py

```diff
@@ -120,7 +120,9 @@ def load_state_dict(self, state):
 
 def _to_saliency_map(capacity, shape=None, data_format='channels_last'):
     """
-    Converts the layer capacity (in nats) to a saliency map (in bits) of the given shape .
+    Converts the layer capacity (in nats) to a saliency map (in bits) of the given shape.
```

Comment:
both pytorch and tensorflow have a `to_saliency_map` which handle the data_format.